readme.md

```diff
@@ -26,7 +35,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+It allows us to make asynchronous requests (typical CRUD requests) on the client side so we can make single page applications.
```

Comment:
:+1:  This is a high level description of what ajax does, we asked for how we use it.  Please review the answer key on the solution branch.